docs/content/about-iota/iota-wallet/how-to/integrate-ledger.mdx

```diff
@@ -26,6 +29,21 @@ To use IOTA Wallet with Ledger, install the IOTA Rebased app on your device thro
 5. Click **Install** to download the IOTA Rebased app to your device.
 6. Your device will show the installation progress.
 
+### Manually (for deprecated devices)
+
+Make sure you have enough space left for an additional app on your device.
```

Comment:
```suggestion
:::tip

Make sure you have enough space left for an additional app on your device.

:::
```

Reply:
Is there a version with a warning instead of a tip?